IntelliTect.TestTools.Selenate/IntelliTect.TestTools.Selenate.Tests/ConditionalWaitTests.cs

```diff
@@ -6,21 +6,28 @@ namespace IntelliTect.TestTools.Selenate.Tests
     public class ConditionalWaitTests
     {
         [Fact]
-        public void CheckActionParamsForTypeChecking()
+        public async void CheckActionParamsForTypeChecking()
```

Comment:
This is not what you want. The test will stop as soon as it hits the `await`. You want the return type of the test to be `async Task`

Reply:
Oh, duh... that was one of the first things you covered with me. Oops. Okay, fixing all occurrences.